tests/src/test/scala/org/apache/openwhisk/core/limits/ActionLimitsTests.scala

```diff
@@ -463,4 +463,35 @@ class ActionLimitsTests extends TestHelpers with WskTestHelpers with WskActorSys
       _.response.result.get.fields("error") shouldBe Messages.memoryExhausted.toJson
     }
   }
+
+  /**
+   * Test that a heavy logging action is interrupted within its timeout limits.
+   */
+  it should "interrupt the heavy logging action within its time limits" in withAssetCleaner(wskprops) {
+    (wp, assetHelper) =>
+      val name = s"NodeJsTestLoggingActionCausingTimeout-${System.currentTimeMillis()}"
+      print(s"\n create action ${name} using api host: ${wskprops.apihost}..")
+      assetHelper.withCleaner(wsk.action, name, confirmDelete = true) { (action, _) =>
+        action.create(
+          name,
+          Some(TestUtils.getTestActionFilename("loggingTimeout.js")),
+          timeout = Some(allowedActionDuration))
+      }
+      val durationMillis = allowedActionDuration + 3.minutes
+      val checkDurationMillis = allowedActionDuration + 1.minutes
+      //val run =
+      //  wsk.action.invoke(name, Map("durationMillis" -> durationMillis.toMillis.toJson, "delayMillis" -> 100.toJson))
+      val run =
+        wsk.action.invoke(name, Map("durationMillis" -> durationMillis.toMillis.toJson, "logDeltaMillis" -> 100.toJson))
```

Comment:
The currently active test code passes parameter `logDeltaMillis` while the test action `loggingTimeout.js` requires a parameter named `delayMillis`.

The commented invocation should be removed.

Reply:
We corrected the parameters and removed the commented code.